networking_ccloud/ml2/agent/common/messages.py

```diff
@@ -211,13 +211,28 @@ def add_iface(self, iface):
 
         return iface
 
-    def get_or_create_iface(self, switchport):
+    def get_iface(self, name):
         if self.ifaces is None:
             self.ifaces = []
```

Comment:
Is order important or could this be a dict for faster access and for not having duplicates? Looks like `.name` is key anyways.